src/fidesops/api/v1/endpoints/health_endpoints.py

```diff
@@ -1,14 +1,62 @@
-from typing import Dict
+import logging
+from typing import Dict, Optional, Union
 
+from alembic import migration, script
 from fastapi import APIRouter
+from redis.exceptions import ResponseError
+from sqlalchemy import create_engine
 
 from fidesops.api.v1.urn_registry import HEALTH
+from fidesops.common_exceptions import RedisConnectionError
+from fidesops.core.config import config
+from fidesops.db.database import get_alembic_config
+from fidesops.util.cache import get_cache
 
 router = APIRouter(tags=["Public"])
 
+logger = logging.getLogger(__name__)
+# stops polluting logs with sqlalchemy / alembic info-level logs
+logging.getLogger("sqlalchemy.engine").setLevel(logging.ERROR)
+logging.getLogger("alembic").setLevel(logging.WARNING)
 
-@router.get(HEALTH, response_model=Dict[str, bool])
-def health_check() -> Dict[str, bool]:
+
+@router.get(HEALTH, response_model=Dict[str, Union[bool, str]])
+def health_check() -> Dict[str, Union[bool, str]]:
     return {
         "healthy": True,
```

Comment:
It'd be great if we could change this response to `"webserver": "healthy"` to standardise the format across the services.

Reply:
good idea, I've updated to `"webserver": "healthy" `